core-bundle/src/Twig/Extension/ContaoExtension.php

```diff
@@ -56,8 +56,11 @@ public function __construct(Environment $environment, TemplateHierarchyInterface
         $escaperExtension->setEscaper('contao_html', [$contaoEscaper, 'escapeHtml']);
         $escaperExtension->setEscaper('contao_html_attr', [$contaoEscaper, 'escapeHtmlAttr']);
 
-        // Use our escaper on all templates in the "@Contao" and "@Contao_*" namespaces
+        // Use our escaper on all templates in the "@Contao" and "@Contao_*"
+        // namespaces, as well as the existing bundle templates we're already
+        // shipping.
         $this->addContaoEscaperRule('%^@Contao(_[a-zA-Z0-9_-]*)?/%');
+        $this->addContaoEscaperRule('%^@Contao(Core|Installation)/%');
```

Comment:
The regex now no longer supports `@Contao_*`. Is this intentional? If so, we should probably adjust the comment.

Reply:
My bad. 🤦‍♂️